activiti-services/activiti-services-query/activiti-services-query-repo/pom.xml

```diff
@@ -1,57 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
@gmalanga are you deleting this on purpose? can you check the status of master? because this might be a problem.